core/src/main/java/org/elasticsearch/script/Script.java

```diff
@@ -149,7 +132,11 @@ public String getLang() {
     }
 
     @Override
-    public final void readFrom(StreamInput in) throws IOException {
+    public final Script readFrom(StreamInput in) throws IOException {
```

Comment:
`readFrom` is going to be removed from the `Writeable` interface, maybe in a few hours. Right now it has a default implementation telling you not to use it. So I'd just skip this bit.